### DISCUSSION TITLE
“Alias” for `eleventyExcludeFromCollections: true`?

### DISCUSSION THREAD

**Discussion by j9t** [11ty/eleventy#1950] [Q&A] [answered]
I probably don’t see the forest for the trees, but what’s the easiest way to create an “alias” for `eleventyExcludeFromCollections: true`?

Background: I use `eleventyExcludeFromCollections: true` quite frequently, like for preparing drafts. It’s such a mouthful, however, that I’d much rather like to use something like `hidden: true` (similar to [Grow](https://grow.io/)).

Am I missing something obvious, is something planned, or is there an elegant way to map names?

**Comment by daflh** [comment, marked as answer]
You can use [Computed Data](https://www.11ty.dev/docs/data-computed/) inside your posts directory like this:
```js
// ./posts/posts.11tydata.js

module.exports = {
  eleventyComputed: {
    eleventyExcludeFromCollections: data => (data.eleventyExcludeFromCollections || data.hidden)
  }
};
```

**Reply by j9t** [reply]
This is great—works like a charm! Thank you!

Follow-up: Is there a way to make this an option for all content? I tried getting it to work in .eleventy.js but haven’t figured that out yet. I’m still getting to know Eleventy.

**Reply by daflh** [reply]
@j9t After doing some experiments on that, I finally figured it out!
Just make an `eleventyComputed.js` file inside your global data folder (probably `_data`), then edit the contents of the file like this:
```js
// ./_data/eleventyComputed.js

module.exports = {
  eleventyExcludeFromCollections: data => (data.eleventyExcludeFromCollections || data.hidden)
};
```

**Reply by j9t** [reply]
Yes, this works! Super-awesome, thank you for the help…!
